curriculum/challenges/english/25-front-end-development/lab-lunch-picker-program/66db529d37ad966480ebb633.md

```diff
@@ -0,0 +1,149 @@
+---
+id: 66db529d37ad966480ebb633
+title: Build a Lunch Picker Program
+challengeType: 14
+dashedName: build-a-lunch-picker-program
+---
+
+# --description--
+
+In this lab, you will create a program that will help you decide what to eat for lunch. You will have two arrays with different food items. You will use the `push`, `pop` and `shift` methods to modify the arrays. You will also use the `Math.random()` method to generate a random number that will be used to select a random meal from the arrays.
```

Comment:
I think this description gives away to much. We should have campers think a little about what methods they will need to use.

```suggestion
In this lab, you'll build a program to assist in choosing lunch options. You'll work with two arrays: one for children's food choices and one for adults. You'll add and remove items from these arrays and then randomly select a meal from each. Finally, you'll log the results to the console.
```